cmd/neofs-adm/internal/modules/morph/internal/types.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // StringifySubnetClientGroupID returns string representation of SubnetClientGroupID using MarshalText.
-// Returns string with message on error.
+// It returns a string with a message on error.
```

Comment:
is it really necessary? i mean i know that it is correct but we have that pattern almost everywhere just to keep decs shorter i guess
will be happy to hear some recommendations about func docs (it always should contain repetitions like "It returns... It panics... It does... It does not..." then)

Reply:
Simple `Returns ...` is quite common https://github.com/golang/go/blob/master/src/cmd/link/internal/loader/loader.go#L752